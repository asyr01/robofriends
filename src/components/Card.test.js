import { shallow, mount, render } from 'enzyme';
import Card from './Card';
import React from 'react';
import renderer from 'react-test-renderer';

it('expect to render Card Component', () => {
  const tree = renderer.create(<Card />).toJSON();
  expect(tree).toMatchSnapshot();
});
