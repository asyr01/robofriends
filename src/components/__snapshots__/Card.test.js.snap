// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card Component 1`] = `
<div
  className="tc grow bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?size=525x525"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
